Fix Docker build on ARM

The Dockerfile contains the step `RUN APP_ENV=prod composer install --no-dev --optimize-autoloader`, which worked fine before on AMD & ARM, but now crashes on ARM only.

The command output is the following:

```txt
...
#42 2.759 Generating optimized autoload files
#42 3.659 51 packages you are using are looking for funding.
#42 3.659 Use the `composer fund` command to find out more!
#42 3.675 > symfony-cmd
#42 3.676 sh: symfony-cmd: not found
#42 3.677 Script symfony-cmd handling the auto-scripts event returned with error code 127
#42 3.677 Script @auto-scripts was called via post-install-cmd
#42 3.677 Plugins have been disabled automatically as you are running as root, this may be the cause of the script failure.
#42 3.677 See also https://getcomposer.org/root
#42 ERROR: process "/bin/sh -c APP_ENV=prod composer install --no-dev --optimize-autoloader" did not complete successfully: exit code: 127
------
 > [production-build 10/10] RUN APP_ENV=prod composer install --no-dev --optimize-autoloader:
2.673  78/78 [============================] 100%
2.759 Generating optimized autoload files
3.659 51 packages you are using are looking for funding.
3.659 Use the `composer fund` command to find out more!
3.675 > symfony-cmd
3.676 sh: symfony-cmd: not found
3.677 Script symfony-cmd handling the auto-scripts event returned with error code 127
3.677 Script @auto-scripts was called via post-install-cmd
3.677 Plugins have been disabled automatically as you are running as root, this may be the cause of the script failure.
3.677 See also https://getcomposer.org/root
------
Dockerfile:331
--------------------
 329 |     COPY composer.lock /var/www/html/composer.lock
 330 |     COPY LICENSE /var/www/html/LICENSE
 331 | >>> RUN APP_ENV=prod composer install --no-dev --optimize-autoloader
 332 |     
 333 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c APP_ENV=prod composer install --no-dev --optimize-autoloader" did not complete successfully: exit code: 127
Error: Process completed with exit code 1.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,7 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Increase feature test timeout from 10 to 15 minutes.
 - Upgrade PHP to 8.3.3, closes #253.
 - Upgrade upstream dependency NGINX Unit to 1.32.0, closes #254.
-- Upgrade dependencies, should fix release process. Closes #257.
+- Upgrade dependencies, closes #257.
+- Add environment variable `COMPOSER_ALLOW_SUPERUSER=1` in release action to fix image build, fixes #259.
 
 ## 0.1.1 - 2024-01-27
 ### Added

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -328,7 +328,7 @@ COPY docker/.env.docker /var/www/html/.env
 COPY composer.json /var/www/html/composer.json
 COPY composer.lock /var/www/html/composer.lock
 COPY LICENSE /var/www/html/LICENSE
-RUN APP_ENV=prod composer install --no-dev --optimize-autoloader
+RUN APP_ENV=prod COMPOSER_ALLOW_SUPERUSER=1 composer install --no-dev --optimize-autoloader
 
 
 

